app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt

```diff
@@ -6,48 +6,48 @@ import uc.edu.klopfsea.groupscheduler.Events.PlannedEvent
 
 class PlannedEventUnitTest {
 
-    var name: String = "Dinner"
-    var date: String = "2021-04-26"
-    var startTime: String = "1700"
-    var plannedEvent: PlannedEvent = PlannedEvent(name, date, startTime)
-    lateinit var invalidPlannedEven: PlannedEvent
+    private var NAME: String = "Dinner"
+    private var DATE: String = "2021-04-26"
+    private var START_TIME: String = "1700"
```

Comment:
These constant strings should be named in all uppercase according to these kotlin conventions, https://kotlinlang.org/docs/coding-conventions.html#property-names